components/dialog/stories/dialog.stories.js

```diff
@@ -14,19 +16,85 @@ export default {
 			type: { name: "string" },
 			table: {
 				type: { summary: "string" },
-				category: "Component",
+				category: "Content",
+			},
+			control: { type: "text" },
+		},
+		header: { 
+			name: "Additional header content",
+			description: "Controls header content",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
 			},
 			control: { type: "text" },
 		},
 		content: { table: { disable: true } },
-		isDismissable: {
-			name: "Dismissable",
+		hasFooter: {
+			name: "Has footer",
+			description: "Adds a footer to the dialog for additional context",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Content",
+			},
+			control: "boolean",
+		},
+		hasCheckbox: {
+			name: "Has checkbox",
+			description: "Adds a checkbox to the footer content.",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Content",
+			},
+			control: { type: "boolean" },
+			if: { arg: "layout", eq: "default" },
+		},
+		footer: {
+			name: "Footer text",
+			description: "Text content of the dialog footer.",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+			control: { type: "text" },
+			if: { arg: "layout", eq: "default" },
+		},
+		size: {
+			name: "Size",
+			type: { name: "string", required: true },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			options: ["small", "medium", "large"],
+			control: "select",
+		},
+		layout: {
+			name: "Layout",
+			type: { name: "string" },
+			defaultValue: "Default",
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+				defaultValue: { summary: "Default" },
+			},
+			// TODO: add the fullscreen and fullscreenTakeover layouts back to options[] once guidance on fullscreen dialogs is determined
```

Comment:
Yes, that's what I was thinking. I made [a followup card](https://jira.corp.adobe.com/browse/CSS-829) a while back for dialog, since we knew that the specs for fullscreen/fullscreentakeover weren't totally solidified for S2 yet.

Reply:
It looks like the takeover dialog designs have been published to Figma. I have some updates according to those designs.

token specs: https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=27417-7447&m=dev
s2 variants/designs: https://www.figma.com/design/Mngz9H7WZLbrCvGQf3GnsY/S2-%2F-Desktop?node-id=61935-5399